contracts/axelar-gateway/src/auth.rs

```diff
@@ -202,3 +202,10 @@ fn validate_signers(env: &Env, weighted_signers: &WeightedSigners) {
         panic_with_error!(env, AuthError::InvalidThreshold);
     }
 }
+
+fn get_signers_epoch(env: &Env, signers_hash: &BytesN<32>) -> u64 {
+    env.storage()
+        .persistent()
+        .get(&DataKey::EpochBySignerHash(signers_hash.clone()))
+        .unwrap_or(0)
```

Comment:
use of default `0` is a bad practice. `get` returns a rich `Option` type which we should use directly. Due to the way `initialize` works, `0` would never be a valid return for epoch by signers hash anyways, since first rotation always assign epoch 1

Reply:
updated.